modules/core/test/load-file/load-image.worker.js

```diff
@@ -0,0 +1,14 @@
+import {loadImage} from '@loaders.gl/core';
+
+if (typeof self !== 'undefined') {
+  /* global self */
+  self.onmessage = (evt) => {
+    const url = evt.data;
+
+    loadImage(url).then(image => {
+      self.postMessage({image}, [image]);
```

Comment:
Does the transferable object argument here ignore non-transferables? Or is `image` always transferable at this point?

Reply:
Hmm I'm not sure. `image` in this context is always `ImageBitmap` which is transferrable.